linter/src/test/java/linter/AppTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package linter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+
+import static org.junit.Assert.*;
+
+public class AppTest {
+
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+
+    @Before
+    public void setUpStreams() {
+        System.setOut(new PrintStream(outContent));
+    }
```

Comment:
Instead of going down this route for your test, just change the return type of your method to `String`, return the String.. then it will be easy to test. Right now this test, really isn't telling us the method is working or not.